modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultCodegen.java

```diff
@@ -2274,6 +2274,10 @@ public CodegenOperation fromOperation(String path,
                 } else if (param instanceof BodyParameter) {
                     bodyParam = p;
                     bodyParams.add(p.copy());
+                    if(definitions != null)
+                    {
```

Comment:
Please update to `if (definitions != null) {` instead to conform to Java style guide.